python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -490,8 +490,16 @@ def terminate_nodes_to_enforce_config_constraints(self, now: float):
         assert self.non_terminated_nodes
         assert self.provider
 
-        last_used = self.load_metrics.last_used_time_by_ip
-        horizon = now - (60 * self.config["idle_timeout_minutes"])
+        ray_nodes_idle_duration_ms_by_ip = (
+            self.load_metrics.ray_nodes_idle_duration_ms_by_ip
+        )
+        now = time.time()
+        last_used = {
+            ip: now - duration
```

Comment:
In `load_metrics.py`, the type hint of `idle_duration_ms` is int. Can we directly subtract `idle_duration_ms` from `time.time()`?